bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/MinPension.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (C) 2010 Nullbyte <http://nullbyte.eu>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.liato.bankdroid.banking.banks;
+
+import android.content.Context;
+import android.text.InputType;
+
+import com.liato.bankdroid.Helpers;
+import com.liato.bankdroid.banking.Account;
+import com.liato.bankdroid.banking.Bank;
+import com.liato.bankdroid.banking.Transaction;
+import com.liato.bankdroid.banking.exceptions.BankChoiceException;
+import com.liato.bankdroid.banking.exceptions.BankException;
+import com.liato.bankdroid.banking.exceptions.LoginException;
+import com.liato.bankdroid.legacy.R;
+import com.liato.bankdroid.provider.IBankTypes;
+
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+import eu.nullbyte.android.urllib.CertificateReader;
+import eu.nullbyte.android.urllib.Urllib;
+
+public class MinPension extends Bank {
+    public MinPension(Context context) {
+        super(context);
+        TAG = "MinPension";
+        NAME = "Min Pension.se";
+        NAME_SHORT = "minpension";
+        BANKTYPE_ID = IBankTypes.MINPENSION;
+        INPUT_TYPE_USERNAME = InputType.TYPE_CLASS_PHONE;
+        INPUT_TYPE_PASSWORD = InputType.TYPE_CLASS_PHONE;
```

Comment:
The intention were to use it, but since the refactoring I don't get completition on R. so I just did as all the other banks, used a static string.